src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -562,21 +561,39 @@ private function registerHttpCacheConfiguration(ContainerBuilder $container, arr
         $loader->load('http_cache_purger.xml');
 
         $definitions = [];
-        foreach ($config['http_cache']['invalidation']['varnish_urls'] as $key => $url) {
-            $definition = new Definition(ScopingHttpClient::class, [new Reference('http_client'), $url, ['base_uri' => $url] + $config['http_cache']['invalidation']['request_options']]);
-            $definition->setFactory([ScopingHttpClient::class, 'forBaseUri']);
 
-            $definitions[] = $definition;
-        }
+        $serviceName = $config['http_cache']['invalidation']['purger'];
+        switch ($serviceName) {
```

Comment:
I think we prefer `match` now

Reply:
The switch case allow us to use varnish by default.